contracts/ERC721M.sol

```diff
@@ -426,7 +415,7 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
         ) revert InvalidCosignSignature();
     }
 
-    function getActiveStageFromTimestamp(uint64 timestamp)
+    function getActiveStageFromTimestamp(uint256 timestamp)
```

Comment:
is this change intentional?

Reply:
Yes; it was intentional. 
Now in some code paths bucket.timestamp is passed to the getActiveStageFromTimestamp and it is uint256. Is it too costly to use uint256 instead of uint64? I can cast bucket.timestamp down to uint64.

Reply:
FYI I updated its type to uint64 and I will update the PR soon.